benchmark/text-generation-inference/benchmark.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+model=${1:-NousResearch/Llama-2-7b-chat-hf}
```

Comment:
Do we really want a default model for the benchmark?

Reply:
It also serves as an explaination of the args that can be passed